File: test/client_code/tests/front/front.js
/*
front/front.js
*/

const { app, component, css } = await use("@/rollo/");
const { Iframe } = await use("assets/tools/iframe", { test: true });

const main = document.getElementById("main");
const frame = document.getElementById("frame");

export default async (state) => {
  //frame.clear(":not([slot])");

  const iframe = Iframe({ name: "front", src: `${origin}/front` });

  frame.append(iframe);


  //
  //
  //iframe.remove()
  
};

File: test/client_code/packages/front/_history/20260426t1330/front.py
def main(use, *args, **kwargs):

    use("@@/assets/")
    mixins = use("@@/mixins")
    Base, Html, On = mixins.Base, mixins.Html, mixins.On
    anvil, console, document, js, log, meta, native, window = (
        use.anvil,
        use.console,
        use.document,
        use.js,
        use.log,
        use.meta,
        use.native,
        use.window,
    )
    component = use("@@/component/")

    use("assets/tools/iframe.css", test=meta.test)
    Iframe = use("assets/tools/iframe", test=meta.test).Iframe

    class front(Base):
        page = True
        persist = True

        def __init__(self, **options):
            Base.__init__(self)
            self.slot="persist"
            iframe = Iframe(dict(name="front", src=f"{meta.origin}/front"))
            self.node.append(iframe)

    return dict(front=front)
